packages/static-build/test/integration-1.test.js

```diff
@@ -45,8 +46,20 @@ const testsThatFailToBuild = new Map([
   ],
 ]);
 
+const fixtureGroups = intoChunks(
+  NUMBER_OF_CHUNKS,
+  fs.readdirSync(fixturesPath)
+);
+const fixturesIndex =
+  parseInt(
+    (module.parent?.filename || __filename).match(/integration-(\d)\./)[1],
```

Comment:
Using `module` or `__filename` is confusing to follow. I suggest exporting a function that the `test/integration-*.test.js` files can require and call where they pass in the appropriate fixture group. It could be something like:

```js
const runTestGroup = require('./integration-tests-by-group');
runTestGroup(1); // 2, 3, ... in the other files
```

Reply:
Updated to remove the module workaround in favor of a setup util in https://github.com/vercel/vercel/pull/7795/commits/a53d099852feb9019791ab523022315b64226ba4